dartsim/src/SDFFeatures_TEST.cc

```diff
@@ -95,16 +109,196 @@ World LoadWorld(const std::string &_world)
   auto world = engine->ConstructWorld(*sdfWorld);
   EXPECT_NE(nullptr, world);
 
-  return *world;
+  return world;
+}
+
+/////////////////////////////////////////////////
+static ignition::math::Pose3d ResolveSdfPose(
+    const ::sdf::SemanticPose &_semPose)
```

Comment:
I guess for a test it doesn't matter, but in general I think it's good to keep local functions as static or put them in anonymous namespaces to avoid name collisions

Reply:
Good point, we can leave this as is :+1: